config/locales/modules/alpha.yml

```diff
@@ -122,5 +122,20 @@ en:
         heading: How to avoid the stuff haters
         body: Hide behind stuff
       1-2-4:
+        heading: End of module
+        body: By viewing this page the training module is marked as complete
```

Comment:
My ask is that demo modules succinct reflect what is going on so recent changes include removing `stuff` and ensuring page content doesn't lie. It'll make frontend testing/demoing easier and less error prone. Please can we ensure any changes to the YAML are semantically accurate.